packages/mesh/src/Mesh.ts

```diff
@@ -314,8 +318,37 @@ export class Mesh<T extends Shader = MeshMaterial> extends Container
 
         renderer.batch.flush();
 
+        let verticesBuffer;
+        let vertices;
+
+        if (this._roundPixels)
+        {
+            verticesBuffer = this.geometry.buffers[0];
+            vertices = verticesBuffer.data;
+
+            this.calculateVertices();
```

Comment:
perhaps this could be an issue as we are modifying the user data and rounding. Maybe better to copy to a roundsVerts buffer?